helpers/PermissionManager.js

```diff
@@ -239,6 +250,7 @@ module.exports.getUserPermissions = async function(username, provider, webstrate
 
 	let permissions = getCachedPermissions(username, provider, webstrateId);
 	if (permissions) {
+		if (await userMustBeLoggedInToWriteAndUserIsAnonymous(username, provider)) permissions = permissions.replace("w", "");
```

Comment:
Although we don't adhere stringently to the rule, we try to keep lines no longer than 100 characters in length. I think readability would benefit from this being on multiple lines.

Also, single quotes over double quotes.

Reply:
I split it up in multiple lines